webui/react/src/omnibar/tree-extension/trees/actions.tsx

```diff
@@ -15,7 +15,7 @@ export const visitAction = (url: string) => (): void => routeToReactUrl(url);
 export const noOp = (): void => undefined;
 export const parseIds = (input: string): number[] => input.split(',').map((i) => parseInt(i));
 
-export const displayHelp = (): void => {
+export const helpContent = (): ReactElement => {
```

Comment:
at this point, can we just make helpContent a full-fledged component instead of a function we call?